src/overlay/FlowControlCapacity.h

```diff
@@ -29,6 +29,7 @@ class FlowControlCapacity
     // Capacity of a connected peer
     uint64_t mOutboundCapacity{0};
     NodeID const& mNodeID;
+    std::optional<uint32_t> mOverlayVersion;
```

Comment:
Instead of making this optional in the parent, why not just add it as a `uint32_t` to `FlowControlByteCapacity`?

Reply:
good idea, fixed.